### DISCUSSION TITLE
Skyline Android Emulator

### DISCUSSION THREAD

**Discussion by DiverseBloom780** [skyline-emu/skyline#347] [Q&A] [answered]
Hey guys, been testing out the skyline emulator on my Snapdragon 855, been running Sonic Mania and Dead cells, it has sound but all I'm getting is a black screen, is there a fix for that?

**Comment by edisionnano** [comment]
Do you use the builds from the FTX1 branch?

**Reply by DiverseBloom780** [reply]
I didn't know anything about using other branches, I will try it out, thanks

**Comment by EliEron** [comment, marked as answer]
Which build of Skyline are you using? Only builds based on the ftx1 branch has GPU support of any kind. The most recent build listed in the Actions tab is not based on ftx1, so if that is the one you are testing then that would explain your issue.

The second most recent build is based on ftx1, so please try switching to that instead.

**Reply by DiverseBloom780** [reply]
Where are those builds at?

**Reply by EliEron** [reply]
Next to the "Discussions" tab near the top of the website there is an "Actions" tab, click on that and you will see a list of recent "runs" each of those contain a build. Each run has the branch it originated from next to it so you can easily see whether it came from ftx1 or not.

Though to save you the trouble I can also provide a direct link to the most recent ftx1 run: [Here](https://github.com/skyline-emu/skyline/actions/runs/1748174707). Just click that link and download the "app-release.apk" artifact.

Do note that you currently have to manually uninstall Skyline from your phone before you can install a different build. Installing over an existing installation does not work.

**Reply by DiverseBloom780** [reply]
Thanks, I will test it out, I am new to github so still trying to figure it out 🙂
